crypto/x509/by_store.c

```diff
@@ -124,7 +124,10 @@ static int by_store_ctrl_ex(X509_LOOKUP *ctx, int cmd, const char *argp,
                 uris = sk_OPENSSL_STRING_new_null();
                 X509_LOOKUP_set_method_data(ctx, uris);
             }
-            return sk_OPENSSL_STRING_push(uris, OPENSSL_strdup(argp)) > 0;
+            char *data = OPENSSL_strdup(argp);
```

Comment:
Declarations go before all code.  so `char *data` needs to go up to below line 120.

Reply:
Thanks, I have submitted a fixup.